### DISCUSSION TITLE
Add support for pandas_profiling

### DISCUSSION THREAD

**Discussion by khuyentran1401** [datapane/datapane#17] [Ideas]
**Is your feature request related to a problem? Please describe.**
It takes quite a bit of code to analyze the data 
**Describe the solution you'd like**
With [pandas_profiling](https://github.com/pandas-profiling/pandas-profiling), an informative report could be created in one line of code. This library becomes increasingly popular. Would be nice if we support this in our Datapane report

**Comment by lanthias** [comment]
Thanks for the suggestion. Would you want this as an extension to the table viewer (probably opening the profile in another view, like a modal), or would you want it as a separate component?

**Comment by khuyentran1401** [comment]
I think either way would work great!
